pkg/envoy/generator/rds.go

```diff
@@ -35,42 +35,39 @@ func (g *EnvoyConfigGenerator) generateRDS(ctx context.Context, proxy *envoy.Pro
 		StatsHeaders(statsHeaders).
 		TrustDomain(trustDomain)
 
-	// Get inbound mesh traffic policy and pass to builder
-	inboundMeshTrafficPolicy := g.catalog.GetInboundMeshTrafficPolicy(proxy.Identity, proxyServices)
-	if inboundMeshTrafficPolicy != nil {
-		routesBuilder.InboundPortSpecificRouteConfigs(inboundMeshTrafficPolicy.HTTPRouteConfigsPerPort)
+	// Get HTTP route configs per port from inbound mesh traffic policy and pass to builder
+	inboundMeshTPHTTPRouteConfigsPerPort := g.catalog.GetInboundMeshTPHTTPRouteConfigsPerPort(proxy.Identity, proxyServices)
+	if inboundMeshTPHTTPRouteConfigsPerPort != nil {
+		routesBuilder.InboundPortSpecificRouteConfigs(inboundMeshTPHTTPRouteConfigsPerPort)
 	}
 
-	// Get outbound mesh traffic policy and pass to builder
-	outboundMeshTrafficPolicy := g.catalog.GetOutboundMeshTrafficPolicy(proxy.Identity)
-	if outboundMeshTrafficPolicy != nil {
-		routesBuilder.OutboundPortSpecificRouteConfigs(outboundMeshTrafficPolicy.HTTPRouteConfigsPerPort)
+	// Get HTTP route configs per port from outbound mesh traffic policy and pass to builder
+	outboundMeshTPHTTPRouteConfigsPerPort := g.catalog.GetOutboundMeshTPHTTPRouteConfigsPerPort(proxy.Identity)
```

Comment:
prefer shorter variable names for variables that aren't used "far away" from the declaration. ie: the lifetime of this variable is ~2 lines.

Reply:
Don't even need these variables TBH if we're getting rid on nil checks.